test/test_models.py

```diff
@@ -605,7 +620,7 @@ def test_classification_model(model_fn, dev):
     _check_jit_scriptable(model, (x,), unwrapper=script_model_unwrapper.get(model_name, None), eager_out=out)
     _check_fx_compatible(model, x, eager_out=out)
 
-    if dev == torch.device("cuda"):
+    if dev == "cuda":
```

Comment:
When debugging, I found out that dev is actually a python str, hence the previous expression `dev == torch.device("cuda")` will always False.

@datumbox is it okay to fix this in this PR, or should I open new PR for this since it is not so related with the PR?